src/transforms/eigenanalysis.jl

```diff
@@ -36,15 +36,21 @@ function pcaproj(λ, V)
 end
 
 function drsproj(λ, V)
-  Λ = Diagonal(sqrt.(λ))
-  S = V * inv(Λ)
-  S, inv(S)
+  λₛ = sqrt.(λ)
+  Λᵢ = Diagonal(1 ./ λₛ)
+  Λₛ = Diagonal(λₛ)
```

Comment:
Can we just use `\Lambda = Diagonal(sqrt.(\lambda))` and the corresponding inverse?